### PR TITLE
[pkp-lib] #2899 Change authorization for custom path navigationMenuItem view op

### PR DESCRIPTION
**Pull request by defstat**

### CODE REVIEW

**Comment by crosfield** [comment]
Unfortunately, these changes do not help in solving the problem. The problem occurs when the user is logged out. Here are screenshots from the current branch right now.

screen shot 2017-10-20 at 19 21 06
screen shot 2017-10-20 at 19 21 44

File: pages/navigationMenu/NavigationMenuItemHandler.inc.php [hunk 1]
```diff
@@ -94,12 +94,6 @@ function view($args, $request) {
 			$contextId = $context->getId();
 		}
 
-		// Ensure that if we're previewing, the current user is a manager or admin.
-		$roles = $this->getAuthorizedContextObject(ASSOC_TYPE_USER_ROLES);
-		if (count(array_intersect(array(ROLE_ID_MANAGER, ROLE_ID_SITE_ADMIN), $roles))==0) {
-			fatalError('The current user is not permitted to preview.');
```
Comment (asmecher) [inline pages/navigationMenu/NavigationMenuItemHandler.inc.php]: It looks like this change just removes the check entirely, correct? Shouldn't previewing rightly be limited to admins and managers?

Comment (defstat) [reply pages/navigationMenu/NavigationMenuItemHandler.inc.php]: @asmecher the preview function has the check. The view function is for displaying the  custom page only.

Comment (asmecher) [reply pages/navigationMenu/NavigationMenuItemHandler.inc.php]: OK, thanks -- will merge.

**Pull request merged**